examples/getting_started/pt/fedavg_script_executor_cifar10.py

```diff
@@ -14,31 +14,41 @@
 
 from src.net import Net
 
-from nvflare import FedAvg, FedJob, ScriptExecutor
+from nvflare import FedJob
+from nvflare.app_common.executors.script_executor import ScriptExecutor
+from nvflare.app_common.workflows.fedavg import FedAvg
+from nvflare.job_config.controller_apps.deep_learning import DeepLearningControllerApp
+from nvflare.job_config.defs import JobTargetType
+from nvflare.job_config.executor_apps.basic import BasicExecutorApp
+from nvflare.job_config.pt.model import PTModel
 
 if __name__ == "__main__":
     n_clients = 2
     num_rounds = 2
     train_script = "src/cifar10_fl.py"
 
     job = FedJob(name="cifar10_fedavg")
+    app = DeepLearningControllerApp()
+    job.to(app, JobTargetType.SERVER)
 
     # Define the controller workflow and send to server
     controller = FedAvg(
         num_clients=n_clients,
         num_rounds=num_rounds,
     )
-    job.to(controller, "server")
+    job.to(controller, JobTargetType.SERVER)
```

Comment:
this is a bit confusing to me the second argument,
sometimes it is "server" or "site-1" sometimes it is `JobTargetType.SERVER`

Reply:
Good point, as target is different from target type. Will use to_server() or "server" instead